src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -182,27 +182,79 @@ button {
     color: red;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/81847092-c8b3a380-955b-11ea-8582-0a69a1df5600.png)

Reply:
Please, fix drag-n-drop component's width for the different breakpoints according to the mockup.

Reply:
fixed